submissions/AnnaStoyano/oop-exercise/index.js

```diff
@@ -0,0 +1,106 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammal{
+   constructor(species,name,gender,legs,friends,saying){
+      this.species = species,
+      this.name = name,
+      this.gender = gender,
+      this.legs = legs,
+      this.friends = friends,
+      this.saying = saying
```

Comment:
Incosistent code style (statements termination). Please fix here and double check everywhere across the code base.